enforcer/src/main/java/org/wso2/micro/gateway/enforcer/security/jwt/JwtTransformerAnnotation.java

```diff
@@ -0,0 +1,18 @@
+package org.wso2.micro.gateway.enforcer.security.jwt;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+/**
+ * Annotation to retrieve data about the jwt transformer class.
+ */
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+
+public @interface JwtTransformerAnnotation {
+    boolean enabled() default false;
+    String name();
+    String issuer();
+}
```

Comment:
Could you please explain what is the annotation I need to use in my custom transformer. I mean a sample annotation if my issuer name is "issuer1"

Reply:
Added comments.